config/docker/chromeos-flash/id_rsa_chromeos_flash

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
It might be worth including the public key file in this PR even if it's not used by the Dockerfile, for completeness. Or is it going to be added as part of a `kci_rootfs` image to run on the Chromebook?

Reply:
It is added as a part of the `kci_rootfs` config config/rootfs/debos/overlays/chromeos-flash/root/.ssh/authorized_keys